front/app/containers/Admin/pages/NewPageForm.tsx

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
+import styled from 'styled-components';
+import { Locale } from 'typings';
+import clHistory from 'utils/cl-router/history';
+
+import GoBackButton from 'components/UI/GoBackButton';
+import PageWrapper from 'components/admin/PageWrapper';
+import PageForm, { FormValues, validatePageForm } from 'components/PageForm';
+import { Formik, FormikProps } from 'formik';
+
+import { FormattedMessage } from 'utils/cl-intl';
+import messages from './messages';
+
+import { createPage } from 'services/pages';
+import { handleAddPageFiles } from 'services/pageFiles';
+
+import { isNilOrError } from 'utils/helperUtils';
+
+import useAppConfigurationLocales from 'hooks/useAppConfigurationLocales';
+
+const PageTitle = styled.h1`
+  width: 100%;
+  font-size: 2rem;
+  margin: 1rem 0 3rem 0;
+`;
+
+export interface Props {}
+
+const NewPageForm = (_props: Props) => {
+  // Still need to handle file saving if we'll use this form.
+  // Also change typing of values parameter to something different (probably FormValues) than 'any'
```

Comment:
Could be removed by now I think? I tested it with a 700 kb PDF and file uploading still works.